### PR TITLE
WIP - Initial version of web pages

### PR DESCRIPTION
**Pull request by dhruv-m-patel**
I have tried putting up landing page and code of conduct page.

Note that there is nothing too fancy in here, just trying to follow basic pages using our branding colors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,7 +19,96 @@
...
+          </div>
+        </div>
+
+        <div class="event-details before-you-come">
+          <div class="container">
+            <h3>Before You Come...</h3>
+            <p>We want to make sure everyone who comes to NodeSchool gets the most out of it, so here are a few things to keep in mind:</p>
+            <p class="que">Never done code before - no HTML or CSS or Excel formulas?</p>
+            <p class="ans">
+              The lessons we work on at NodeSchool are going to be pretty difficult.
```
Comment (normrider2001) [inline index.html]: I don't like this line.  I think we should be more encouraging.

File: codeofconduct.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+          <p>Participants asked to stop any harassing behavior are expected to comply immediately.</p>
+          <p>Sponsors are also subject to the anti-harassment policy. In particular, sponsors should not use sexualized images, activities, language, or other material. Sponsor staff (including volunteers) should not wear overtly sexualized clothing/uniforms/costumes or otherwise create a sexualized environment.</p>
+          <p>If a participant engages in harassing behavior, the organizers may take any action they deem appropriate, including warning the offender or expulsion (with no refund where applicable).</p>
+          <p>If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact an event volunteer or organizer immediately.</p>
+          <p>If you can't easily locate an event volunteer or organizer or if you are uncomfortable doing so, you can contact the lead organizer, Jason Rhodes, directly and discretely using any of the methods above "Need help?" section.</p>
```
Comment (normrider2001) [inline codeofconduct.html]: Remove the name Jason Rhodes replace with my name if you want.

**Review by normrider2001 — CHANGES_REQUESTED** [review]
We need to change this guy's name.   I also included the code of conduct on a different readme.

File: codeofconduct.html (hunk 2 above)
Comment (dhruv-m-patel) [reply codeofconduct.html]: So sorry I didn't even pay attention to that one while putting copied content! I will remove it.

**Comment by dhruv-m-patel** [comment]
Content has been updated.

**Review by normrider2001 — APPROVED** [review]

**Pull request merged**